manifests/broker/service.pp

```diff
@@ -23,6 +26,12 @@
       content => template('kafka/init.erb'),
     }
 
+    if $::osfamily == 'RedHat' and $::operatingsystemmajrelease == '7' {
```

Comment:
It's in stdlib:

https://github.com/puppetlabs/puppetlabs-stdlib/blob/master/lib/facter/service_provider.rb